src/app/routes/connector-ui/policy-definition-page/new-policy-dialog/new-policy-dialog.component.html

```diff
@@ -25,27 +25,57 @@ <h1 mat-dialog-title>Create New Policy</h1>
         </mat-option>
       </mat-select>
     </mat-form-field>
+    <!-- Time Interval Type -->
+    <ng-container *ngIf="form.policyType === 'Time-Period-Restricted'">
+      <mat-form-field color="accent">
+        <mat-label>Date Selection Criteria</mat-label>
+        <mat-select [formControl]="form.group.controls.dateSelectionType">
+          <mat-option value="Start-Date"> Start Date (open-ended) </mat-option>
+          <mat-option value="Date-Range"> Date Range</mat-option>
+        </mat-select>
+      </mat-form-field>
```

Comment:
In my opinion this select is hard to name / explain.

Maybe we could replace it with a checkbox like we do for things in the asset data source form?

And name it "Open Ended" or something like that?